openedx/core/djangoapps/xblock/runtime/runtime.py

```diff
@@ -343,6 +344,7 @@ def service(self, block: XBlock, service_name: str):
         elif service_name == 'error_tracker':
             return make_error_tracker()
         elif service_name == 'video_config':
+            from openedx.core.djangoapps.video_config.services import VideoConfigService
```

Comment:
Why is this import here instead of at the top of the file where all the other services are imported?  If there is no circular dependency, this should be moved up.